glide.yaml

```diff
@@ -28,6 +28,8 @@ import:
   version: ^1
 - package: github.com/stretchr/testify
   version: master
+- package: go.uber.org/fx
```

Comment:
Actually, I think we can make this a testImport. @vivekrsharma can you move this to the `testImport` section?